charts/egeria-platform/scripts/config-view1.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+export EGERIA_VIEW_SERVER=view1
+
+# Set the URL root
+echo -e '\n\n > Setting view server URL root:\n'
+curl -f -k --verbose --basic admin:admin -X POST \
+  "${EGERIA_LOCAL_ENDPOINT}/open-metadata/admin-services/users/${EGERIA_USER}/servers/${EGERIA_VIEW_SERVER}/server-url-root?url=${EGERIA_ENDPOINT}"
+
+# Setup the event bus
+echo -e '\n\n > Setting up event bus:\n'
+curl -f -k --verbose --basic admin:admin \
+  --header "Content-Type: application/json" \
+  "${EGERIA_LOCAL_ENDPOINT}/open-metadata/admin-services/users/${EGERIA_USER}/servers/${EGERIA_VIEW_SERVER}/event-bus" \
+  --data '{"producer": {"bootstrap.servers": "'"${EGERIA_KAFKA_ENDPOINT}"'"}, "consumer": {"bootstrap.servers": "'"${EGERIA_KAFKA_ENDPOINT}"'"} }'
+
+# Set as view server
+echo -e '\n\n > Set as view server:\n'
+curl -f -k --verbose --basic admin:admin -X POST \
+  "${EGERIA_LOCAL_ENDPOINT}/open-metadata/admin-services/users/${EGERIA_USER}/servers/${EGERIA_VIEW_SERVER}/server-type?typeName=View%20Server"
+
+# Configure the view server cohort membership
+echo -e '\n\n > configuring cohort membership:\n'
+curl -f -k --verbose --basic admin:admin -X POST \
+  "${EGERIA_LOCAL_ENDPOINT}/open-metadata/admin-services/users/${EGERIA_USER}/servers/${EGERIA_VIEW_SERVER}/cohorts/${EGERIA_COHORT}"
+
+# Configure the view services
+echo -e '\n\n > Setting up Glossary Author:\n'
+curl -f -k --verbose --basic admin:admin \
+   --header "Content-Type: application/json" \
+   "${EGERIA_LOCAL_ENDPOINT}/open-metadata/admin-services/users/${EGERIA_USER}/servers/${EGERIA_VIEW_SERVER}/view-services/glossary-author" \
+   --data @- <<EOF
+{
+  "class": "ViewServiceConfig",
+  "omagserverPlatformRootURL": "${EGERIA_OMAG_SERVER_URL}",
+  "omagserverName" : "${EGERIA_OMAG_SERVER_NAME}"
+}
+EOF
+
+echo -e '\n\n > Setting up TEX:\n'
+curl -f -k --verbose --basic admin:admin \
+  --header "Content-Type: application/json" \
+  "${EGERIA_LOCAL_ENDPOINT}/open-metadata/admin-services/users/${EGERIA_USER}/servers/${EGERIA_VIEW_SERVER}/view-services/tex" \
+  --data @- <<EOF
+{
+  "class":"IntegrationViewServiceConfig",
```

Comment:
As this chart is targetted at the user wanting to deploy multiple servers, I think we need to highlight the change needed in the view service config to configured desired servers. Just a one liner and reference back to the egeria docs is probably sufficient?